src/test/java/gov/nih/nci/evs/api/service/CaptchaServiceTest.java

```diff
@@ -36,12 +36,14 @@ public class CaptchaServiceTest {
   @InjectMocks private CaptchaService captchaService;
 
   /** Setup before each test. */
+  @SuppressWarnings("resource")
```

Comment:
@bcarlsenca I've seen these annotations in our tests, but I'm not sure what these do. Are they for suppressing certain warnings in the logs? It might be good to review this with our best practices for new code implementations.